modules/src/wiki.py

```diff
@@ -1,14 +1,20 @@
 import wikipedia
+import string
 from templates.text import TextTemplate
 
 def process(input, entities):
     output = {}
     try:
         query = entities['wikipedia_search_query'][0]['value']
-        data = wikipedia.summary(query)
+        data = wikipedia.page(query)
+        template = TextTemplate()
+        printable = set(string.printable)
+        s = filter(lambda x : x in printable, data.summary)
```

Comment:
![](https://cloud.githubusercontent.com/assets/2833845/15002985/af34dd8a-11c4-11e6-94d4-b810b494ad8f.PNG)
The summary seems to be rendering correctly, so we can remove the filtering of string.printable.
And the ["text"] is not required; I had made some changes so that get_message() returns the actual message sent to the Messenger API (it includes the 'text' in the json)